RELEASE.md

```diff
@@ -0,0 +1,13 @@
+# Release Process
+
+The process of creating a release is described in this document. Replace `X.Y.Z` with the version to be released.
+
+## 1. Create a `vX.Y.Z` tar 
+
+Manually run the `Create tag` action. This will create a git tag and update the `VERSION` file 
+
+## 2. Create a [new release](https://github.com/fybrik/json-schema-generator/releases/new) 
```

Comment:
This link redirects me to `https://github.com/`. Maybe it will open the desired page for users with "creating a new release" permissions.

Reply:
yes, probably, can you try to open the "new release page"

Reply:
I don't have such an option in `releases`.

Reply:
I see